src/jmcomic/jm_option.py

```
@@ -298,7 +298,7 @@ def construct(cls, origdic: Dict, cover_default=True) -> 'JmOption':
 
         # log
         log = dic.pop('log', True)
-        if log is False:
+        if not log:
             disable_jm_log()
```

Ruff (0.15.7)

[error] `disable_jm_log` may be undefined, or defined from star imports

(F405)